test/test-2.txt

```diff
@@ -1,2 +0,0 @@
-○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○──○
```

Comment:
Is this intentional?

Reply:
Oops!  I was using that directory for my own test-3.txt/etc., and thought one of my files had accidentally been included .  Re-added.  Let me know and I can squash all this again.